### PR TITLE
Validate Huber delta is greater than 0

### PR DESCRIPTION
**Pull request by kaivalya-cyber**
A non-positive `delta` in `keras.losses.Huber` silently produces degenerate loss values instead of an error. For example, `delta=0` collapses the linear branch to zero, so every sample's loss becomes the same wrong value.

This PR raises a `ValueError` at construction time when `delta` is a Python number `<= 0` (backend tensors are skipped and validated at runtime), mirroring the validation that existed in TF-Keras 2 and the `q` validation in `GeneralizedCrossEntropy`.

Changes:
- `keras/src/losses/losses.py`: validate `delta > 0` in `Huber.__init__`.
- `keras/src/losses/losses_test.py`: add `test_invalid_delta`.

Checks: `ruff check`, `ruff format --check`, `compileall`, and `git diff --check` pass locally. The focused pytest is blocked locally because `optree` is not installed.

## Contributor Agreement

**Please review our [AI-Assisted Contribution Policy](../CONTRIBUTING.md#ai-assisted-contribution-policy) and check all boxes below before submitting your PR for review:**

- [x] I am a human, and not a bot.
- [x] I will be responsible for responding to review comments in a timely manner.
- [x] I will work with the maintainers to push this PR forward until submission.

**Note:** Failing to adhere to this agreement may result in your future PRs no longer being reviewed.

### CODE REVIEW

File: keras/src/losses/losses.py [hunk 1]
```diff
@@ -343,6 +343,11 @@ def __init__(
         name="huber_loss",
         dtype=None,
     ):
+        if not ops.is_tensor(delta) and delta <= 0:
```
Comment (hertschuh) [inline keras/src/losses/losses.py]: Actually, `delta` should just be a float, I don't think we should allow tensors.

So just use `if not isinstance(delta, float) or delta <= 0`

**Review by hertschuh — CHANGES_REQUESTED** [review]

**Comment by kaivalya-cyber** [comment]
@hertschuh Done. delta now must be a Python float greater than 0 — the tensor escape hatch is removed and any other type (e.g. int or NumPy scalars) raises ValueError at construction. Added test coverage for both non-float cases. Pushed in e070724c7; Ruff, formatting, compilation, and diff checks pass locally, and the focused pytest runs in CI.

**Review by hertschuh — APPROVED** [review]
Thanks!
